general_rules/Scenario.tex

```diff
@@ -176,7 +176,6 @@ \subsection{Predefined Rooms and Locations}
 \label{rule:scenario_locations}
 
 Some tests in the RoboCup@Home league involve a \PredefinedLocation{} where people or objects can be found.
-Each \PredefinedLocation{} has an assigned \LocationClass{} (for instance, a \textit{Couch} and an \textit{Arm Chair} belong to the \textit{Seating} class).
```

Comment:
Any reason to remove this definition?

Reply:
It has not been provided in past tournaments. There is also no mention of it in the rest of the rulebook. So this is a definition without use

Reply:
Fair